src/databricks/labs/dqx/installer/install.py

```diff
@@ -82,6 +85,10 @@ def installation(self):
         try:
             return self.product_info.current_installation(self.workspace_client)
```

Comment:
this will fail on group assigned cluster with Error: 404 since it is using get me api

Reply:
updated